### ISSUE TITLE
Flaky build: The type or namespace name 'Loader' does not exist in the namespace 'System.Runtime'

### ISSUE THREAD

**Issue by khyperia** [dotnet/roslyn#22602]
```
Hosting/AssemblyLoader/CoreAssemblyLoaderImpl.cs(7,22): error CS0234: The type or namespace name 'Loader' does not exist in the namespace 'System.Runtime' (are you missing an assembly reference?)
```

Example PRs with repro:

* https://github.com/dotnet/roslyn/pull/22151
* https://github.com/dotnet/roslyn/pull/22424
* https://github.com/dotnet/roslyn/pull/22487
* https://github.com/dotnet/roslyn/pull/22527

Retesting usually resolves it.

**Comment by jasonmalinowski** [comment]
This suspiciously has gone away.

**Comment by granadacoder** [comment]
For future readers.  This may be a DotNetCore 3.1 only function.  And not available in 2.1, or Standard2.0.  That was my issue.  I was trying to use this function in a Standard2.0 library..and that wasn't playing nicely.
